internal/trigger/http/rest.go

```diff
@@ -43,9 +43,14 @@ type Trigger struct {
 }
 
 // Initialize initializes the Trigger for logging and REST route
-func (trigger *Trigger) Initialize(appWg *sync.WaitGroup, appCtx context.Context) (bootstrap.Deferred, error) {
+func (trigger *Trigger) Initialize(appWg *sync.WaitGroup, appCtx context.Context, background <-chan types.MessageEnvelope) (bootstrap.Deferred, error) {
 	logger := trigger.EdgeXClients.LoggingClient
 
+	if background != nil {
+		// Is error sufficient here or should we block startup?
```

Comment:
Yes, I think this should block start-up as the developer is trying to use in away not intended. Returning error will give immediate feedback and avoid head scratching when it isn't working they why they thought it would.